data/dashboard.yml

```diff
@@ -41,6 +41,17 @@
         - name: Repositories
           url: /repos.html
           description: All the applications and other repositories we use to build GOV.UK.
+    - name: GOV.UK manuals
+      sites:
+        - name: 2nd-line Tech Support manuals
+          url: /manual/2nd-line.html
+          description: The documentation for GOV.UK's Second-line Technical Support team.
+        - name: Developer manuals
+          url: /manual.html
+          description: The contents page of GOV.UK's developer manuals.
+        - name: GOV.UK Kubernetes manuals
+          url: https://govuk-k8s-user-docs.publishing.service.gov.uk
+          description: The developer manuals for GOV.UK's Kubernetes clusters.
```

Comment:
Is it worth adding a note like "we intend to consolidate these into the Developer Docs"?